common/views/components/Pagination/Pagination.tsx

```diff
@@ -1,23 +1,26 @@
 import { FunctionComponent } from 'react';
-import { font } from '../../../utils/classnames';
+import { classNames, font } from '../../../utils/classnames';
 import Control from '../Buttons/Control/Control';
 import Space from '../styled/Space';
 import Rotator from '../styled/Rotator';
 import { arrow } from '@weco/common/icons';
 import styled from 'styled-components';
+import { LinkProps } from '../../../model/link-props';
 
-export type Props = {
-  totalResults: number;
+type PageChangeFunction = (event: Event, page: number) => Promise<void>;
+
+export type PaginatedResultsProps = {
   currentPage: number;
   totalPages: number;
-  prevPage?: number;
-  nextPage?: number;
-  prevQueryString?: string;
-  nextQueryString?: string;
-  range?: {
-    beginning: number;
-    end: number;
-  };
+};
+
+export type Props = {
+  paginatedResults: PaginatedResultsProps;
+  paginationRoot: LinkProps;
+  hideMobilePagination?: boolean;
+  disabled?: boolean;
+  onPageChange?: PageChangeFunction;
+  showPortal?: boolean;
```

Comment:
These extra parameters all come from the catalogue Paginator, but in the content app they're unused.